packages/ui/src/documents/Nfts.tsx

```diff
@@ -92,37 +85,38 @@ export const Nfts: FunctionComponent<Props> = (props) => {
     }
   };
 
-  const transferNFT = async (data: TransferNftRequest) => {
-
-    onAsyncStart('Transferring NFT');
-
-    try {
-      onAsyncComplete((await httpClient.nfts.tranfer(
-        data,
-      )).data);
-    } catch (e) {
-      onAsyncError(e, 'Failed to transfer NFT');
-    }
-  };
-
   const openMintModal = () => {
     setState({ mintModalOpened: true });
   };
 
-  const openTransferModal = (selectedNft: CoreapiNFT) => {
-    setState({
-      selectedNft,
-      transferModalOpened: true,
-    });
-  };
-
   const closeModal = () => {
     setState({
       mintModalOpened: false,
       transferModalOpened: false,
     });
   };
 
+  const assembleDeepLink = (tokenId: string, registry: string, tinlakePool: string,) => {
+    const address = `${tinlakePool}/loans/issue?tokenId=${tokenId}&registry=${registry}`
```

Comment:
would use the url constructor to create URLs https://javascript.info/url

Reply:
i think normally I would agree with this, but in this case the params are just going to be addresses/numbers for the tokenid so seems to be unnecessary extra overhead to encode/decode the URIs